decidim-bulletin_board-app/app/services/voting_scheme/electionguard.rb

```diff
@@ -2,11 +2,11 @@
 
 require "pycall/dict"
 require "pycall"
-EGBB = PyCall.import_module("decidim.electionguard.bulletin_board")
+EGBB = PyCall.import_module("bulletin_board.electionguard.bulletin_board")
 
 module VotingScheme
   # An implementation of the ElectionGuard voting scheme, using PyCall to execute the ElectionGuard python code
```

Comment:
```suggestion
  # An implementation of the Electionguard voting scheme, using PyCall to execute the Electionguard python code
```

Reply:
Thanks for the feedback, but my idea is to keep using ElectionGuard as the name of the project, even when the Ruby classes or modules will be called Electionguard to prevent the `_`.